src/main/java/seedu/address/storage/JsonSerializableAddressBook.java

```diff
@@ -22,18 +25,22 @@
 
     public static final String MESSAGE_DUPLICATE_PROPERTY = "Property list contains duplicate property(s).";
     public static final String MESSAGE_DUPLICATE_BUYER = "Buyer list contains duplicate buyer(s).";
+    public static final String MESSAGE_DUPLICATE_MATCH = "Match list contains duplicate match(es).";
+
 
     private final List<JsonAdaptedProperty> properties = new ArrayList<>();
     private final List<JsonAdaptedBuyer> buyers = new ArrayList<>();
-
+    private final List<JsonAdaptedMatch> matches = new ArrayList<>();
```

Comment:
by adding this, matches are going to be visible to the user via the JSON file, I think this might be an issue if we don't end up persisting the data.